src/lightning/pytorch/trainer/trainer.py

```diff
@@ -1070,6 +1070,14 @@ def __setup_profiler(self) -> None:
         self.profiler._lightning_module = proxy(self.lightning_module)
         self.profiler.setup(stage=self.state.fn, local_rank=local_rank, log_dir=self.log_dir)
 
+    def print(self, *args: Any, **kwargs: Any) -> None:
+        """Print something only on the first process.
```

Comment:
```suggestion
        """Print the arguments to standard output. When running on multiple devices on a single node, this method will only print from one process to avoid redundant outputs. When running across multiple nodes, this method will print from one process in each node.
```